### PR TITLE
Added few useful modification

### PR DESCRIPTION
**Pull request by blackPantherOS**
Hi,
We made small modification on last Winetricks:
- shebang fix
- added distrospecific correct installation message
- fix wine64 binary detect
- added PyDialog (Qt5 based but desktop independ) support
- added first icon displaying for better looking test
Check and merge if like to you

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
+#!/usr/bin/sh
+
```
Comment (austin987) [inline src/winetricks]: No.

File: src/winetricks [hunk 2]
```diff
@@ -256,8 +270,10 @@ w_warn()
 
     case $WINETRICKS_GUI in
         zenity) $WINETRICKS_GUI "$_W_timeout" --error --title=winetricks --text="$(echo "$@" | sed 's,\\\\,\\\\\\\\,g')";;
-        kdialog) $WINETRICKS_GUI --title winetricks --error "$@" ;;
-        none) ;;
+        pydialog|kdialog) $WINETRICKS_GUI --title winetricks --error "$@" ;;
+        none) 
+        [ -x $(which notify-send) ]&& notify-send  "Winetricks Warn"  "$@"
```
Comment (austin987) [inline src/winetricks]: Indent

File: src/winetricks [hunk 3]
```diff
@@ -7,7 +8,20 @@
 # Name of this version of winetricks (YYYYMMDD)
 # (This doesn't change often, use the sha256sum of the file when reporting problems)
 WINETRICKS_VERSION=20180815-next
-
+# added correct install message on any other systems from Ubuntu
+if [ -f /etc/blackPanther-release ];then
+    INSTALLCMD="installing"
+    GUIINSTALLCMD="app-helper"
+elif [ -f /etc/fedora-release -o  -f /etc/redhat-release ];then
```
Comment (austin987) [inline src/winetricks]: Don't use -o, see shellcheck output.

File: src/winetricks [hunk 4]
```diff
@@ -7,7 +8,20 @@
...
+    INSTALLCMD="dnf install"
+    GUIINSTALLCMD="pkcon install"
+elif [ -f /etc/mandrake-release -o -f /etc/mandriva-release -o -f /etc/mangeia-release -o -f /etc/rosa-release ];then
+    INSTALLCMD="urpmi"
+    GUIINSTALLCMD="gurpmi2"
+else 
+    INSTALLCMD="sudo apt-get install"
```
Comment (austin987) [inline src/winetricks]: Why sudo here and not elsewhere?

File: src/winetricks [hunk 5]
```diff
@@ -279,8 +295,10 @@ w_warn_cancel()
     # Zenity has no cancel button, but will set status to 1 if you click the go-away X
     case $WINETRICKS_GUI in
         zenity) $WINETRICKS_GUI "$_W_timeout" --error --title=winetricks --text="$(echo "$@" | sed 's,\\\\,\\\\\\\\,g')";;
-        kdialog) $WINETRICKS_GUI --title winetricks --warningcontinuecancel "$@" ;;
-        none) ;;
+        pydialog|kdialog) $WINETRICKS_GUI --title winetricks --warningcontinuecancel "$@" ;;
+        none)
+        [ -x $(which notify-send) ]&& notify-send  "Winetricks Warn"  "$@"
```
Comment (austin987) [inline src/winetricks]: Indent

File: src/winetricks (hunk 4 above)
Comment (blackPantherOS) [reply src/winetricks]: > `Why sudo here and not elsewhere?

This line just copied from your source. We don't use sudo

**Pull request closed without merging**